src/transform/RedLinks.js

```diff
@@ -12,16 +12,15 @@ const configureRedLinkTemplate = (span, anchor) => {
 }
 
 /**
- * Finds red links in a document or document fragment.
- * @param {!(Document|DocumentFragment)} content Document or fragment in which to seek red links.
```

Comment:
Please add a release note for this API-breaking change.

Reply:
Ok if I update the changelog separately with note for this and https://github.com/wikimedia/wikimedia-page-library/pull/122, https://github.com/wikimedia/wikimedia-page-library/pull/123 and 
https://github.com/wikimedia/wikimedia-page-library/pull/124 ?

Reply:
That way I can just cut one build having all 4 updates.

Reply:
:+1: Sounds good.